packages/client/src/v2-events/features/events/components/FormWizard.tsx

```diff
@@ -79,12 +79,9 @@ export const FormWizard = ({
             >
               {continueButtonText}
             </Button>
-
-            {showReviewButton && (
```

Comment:
Is the `showReviewButton` being used for something else? If not can we remove it then?